lib/site-builder.js

```diff
@@ -43,12 +43,12 @@ function SiteBuilder(opts, branch, commandRunner, jekyllHelper,
   }
 }
 
-SiteBuilder.prototype.generateBuildDone = function(done) {
+SiteBuilder.prototype.finishBuild = function() {
   var builder = this,
```

Comment:
Sorry if I've missed this earlier.

Usual JS convention dictates to use the var name `self` for local scoping `this`. Is that possible here, or would it be too much work?